### PR TITLE
Remove me from owners ☹️

### PR DESCRIPTION
**Pull request by tetchel**
I keep getting my review requested on this project I'm no longer a part of. 😭

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -14,7 +14,6 @@ reviewers:
 - chetan-rns
 - mitchharpur
 - keithchong
-- tetchel
 - amitkrout
 - jaideepr97
 - gaganhegde
```
Comment (wtam2018) [inline OWNERS:19]: Let's remove jaideepr97 and gaganhegde, too.

Comment (tetchel) [reply OWNERS:19]: ☹️

**Comment by sbose78** [comment]
You may still review PRs @tetchel ;)

**Comment by tetchel** [comment]
👋

**Pull request merged**